src/errors/ilp-error.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const BaseError = require('five-bells-shared').BaseError
```

Comment:
Rather than extending all of them directly in the code, it would probably be better to have an ILP error module as suggested in https://github.com/interledgerjs/ilp-plugin-bells/issues/83